src/package/store.rs

```diff
@@ -145,10 +148,12 @@ impl PackageStore {
         &self,
         manifest: &Manifest,
     ) -> miette::Result<crate::validation::Violations> {
-        let pkg_path = self.proto_path();
-        let source_files = self.collect(&pkg_path, false).await;
+        let root_path = self.proto_vendor_path();
```

Comment:
this looks wrong to me – im 99.9% confident, that we dont want to lint the protocol buffers in `proto/vendor` but the ones in `proto/proto.*`!

Reply:
@xfbs what do you think?

Reply:
Ah, I get what you mean. To be honest — it should not matter, right? As long as they are in sync?

Reply:
I think we have to make the ``collect`` a bit more accessible...
What we do here is, we validate the files in ``proto/vendor/<package>/*.proto`` with the include path ``proto/vendor/``. Since the validation tries to compile the package, we have to do it there in order to resolve the absolute import paths.